assets/js/components/Sectors.vue

```diff
@@ -237,18 +247,40 @@ export default Vue.extend({
       label.moveToBack();
 
       $this._labels
-      .data($this.getRoot().descendants().slice(1))
-        .style("display", (d) => { return d.data.enabled ? "inline" : "none"; })
-        .style("opacity", function(d) { return !d.data.enabled ? 0 : this.opacity; })
+       .data($this.getRoot().descendants().slice(1))
+       .on("click", function(d){
```

Comment:
there should be 2 "click" functions, lvl1->lvl2 (what was there already), and lvl2->lvl1.
you can come up with the naming.